extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -28,15 +30,71 @@ class AmpStickyAd extends AMP.BaseElement {
   isLayoutSupported(layout) {
     return layout == Layout.NODISPLAY;
   }
-
-  /** @override */
+  /**
+   * @override
+   * @this {undefined}  // Make linter happy
```

Comment:
that shouldn't be needed. @zhouyx i'm guessing this was copied from amp-ad? that fix was for closure compiler warnings